portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsBooleanField.testcase

```diff
@@ -0,0 +1,413 @@
+	@priority = "5"
+	test CanBeSubmitted {
+		FormsAdmin.addForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Boolean");
+
+		FormFields.editText(
+			fieldName = "label",
+			fieldValue = "Boolean 1");
+
+		Form.closeSidebar();
+
+		Form.gotoAddField(
+			fieldPositionNumber = "1",
+			fieldType = "Boolean");
+
+		FormFields.editText(
+			fieldName = "label",
+			fieldValue = "Boolean 2");
+
+		FormFields.viewBooleanField(fieldValue = "Boolean 1");
+
+		FormFields.viewBooleanField(fieldValue = "Boolean 2");
+
+		FormViewBuilder.gotoFirstFieldset(fieldLabel = "Boolean 1");
+
+		FormFields.enableSwitch(fieldName = "Show as a Switch");
+
+		FormFields.viewSwitch(fieldName = "Boolean 1");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormFields.viewSwitch(fieldName = "Boolean 1");
+
+		FormFields.viewCheckboxNotChecked(fieldName = "Boolean 2");
+
+		FormFields.enableCheckbox(fieldName = "Boolean 2");
+
+		FormPortlet.submitSuccessfully();
+
+		Navigator.openURL();
+
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoViewEntries();
+
+		FormsAdmin.viewFormTableEntry(
+			fieldLabel = "Boolean 1",
+			fieldValue = "False");
+
+		FormsAdmin.viewFormTableEntry(
+			fieldLabel = "Boolean 2",
+			fieldValue = "True");
+	}
+
+	@description = "Verify that a form with the boolean field, its help text can be added"
+	@priority = "5"
+	test HelpTextCanBeAdded {
+		FormsAdmin.addForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Boolean");
+
+		FormFields.viewBooleanField(fieldValue = "Boolean");
+
+		FormFields.editHelpTextField(fieldHelpText = "Test Help Text");
+
+		FormFields.viewFieldHelpText(
+			fieldHelpText = "Test Help Text",
+			fieldName = "Boolean");
+
+		FormViewBuilder.assertHelpTextPresent(
+			fieldName = "Boolean",
+			fieldValue = "Test Help Text");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormViewBuilder.assertHelpTextPresent(
+			fieldName = "Boolean",
+			fieldValue = "Test Help Text");
+	}
+
+	@description = "Verify that a form with the boolean field can be edited"
+	@priority = "5"
+	test LabelCanBeEdited {
+		FormsAdmin.addForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Boolean");
+
+		FormFields.viewBooleanField(fieldValue = "Boolean");
+
+		FormFields.editText(
+			fieldName = "label",
+			fieldValue = "Test Boolean Field");
+
+		FormFields.viewBooleanField(fieldValue = "Test Boolean Field");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormFields.viewBooleanField(fieldValue = "Test Boolean Field");
+	}
+
+	@description = "Verify that a form with the boolean field, its reference can be edited"
```

Comment:
@rafaellajordao, please visit the test's description because it seems to be inconsistent with the test name.